package.json

```diff
@@ -50,8 +50,8 @@
     "test": "pnpm run -C examples/sveltekit-ts test"
   },
   "peerDependencies": {
-    "@sveltejs/kit": "^1.3.1",
-    "vite-plugin-pwa": ">=0.16.7 <1"
+    "@sveltejs/kit": "^1.3.1 || ^2.0.0",
```

Comment:
sveltekit 2 requires vite5, sveltekit 1 requires vite4

Reply:
It is a dev dependency, Vite 4 and Vite 5 don't change plugin types, it should be compatible with Rollup 3 and 4.

I've tested this PR also with kit 1 and tests green:
- extract example from main branch to a new folder
- remove `@vite-pwa/sveltekit` from dev dependencies (using workspace protocol)
- add vite 4.5.1, vite-plugin-pwa, workbox-window, workbox-precaching and workbox-routing as dev dependencies
- `pnpm build && pnpm pack` in this PR, copy tgz to the new project folder
- install the plugin using file protocol: `pnpm add -D ./vite-pwa-sveltekit-0.2.10.tgz`
- tests green: `pnpm test`